### PR TITLE
Remove unneeded options in shoppingcart.proto

### PR DESCRIPTION
**Pull request by vkorenev**
See also cloudstateio/samples-ui-shoppingcart#12

### CODE REVIEW

**Comment by vkorenev** [comment]
@coreyauger

File: shopping-cart/shoppingcart.proto [hunk 1]
```diff
@@ -38,25 +37,14 @@ message Cart {
 
 service ShoppingCart {
     rpc AddItem(AddLineItem) returns (google.protobuf.Empty) {
-        option (google.api.http) = {
-            post: "/com.example.shoppingcart.ShoppingCart/cart/{user_id}/items/add",
-            body: "*",
-        };
-        option (.cloudstate.eventing).in = "items";
+        option (google.api.http).post = "/com.example.shoppingcart.ShoppingCart/cart/{user_id}/items/add";
```
Comment (justinhj) [inline shopping-cart/shoppingcart.proto:40]: I'm not familiar with why we no longer need the body option...

Comment (vkorenev) [reply shopping-cart/shoppingcart.proto:40]: I wonder why did it was needed in the first place. Anyway, everything seems to work fine without it.

Comment (vkorenev) [reply shopping-cart/shoppingcart.proto]: @justinhj, `body: "*"` is needed for HTTP transcoding. However, HTTP is not used at the moment. Anyway, I returned it back.

**Pull request closed without merging**